### ISSUE TITLE
Disk Storage usage data isn't accurate

### ISSUE THREAD

**Issue by poundbangbash** [munkireport/munkireport-php#277]
I just updated a server to 2.5.2.1323 by moving aside the existing installation, copying back my config & db, changing the config.php to allow migrations, and all is running fine.
I then created a new installer from the 2.5.2 server and distributed to two test clients via munki.  Both installed fine and show no errors, however now when looking at both clients on the server the Storage section shows:

Disk Size: 2.1 GB
Used: 0 B
Free: 2.1 GB
SMART status: verified

However that is far from correct.  

The data from one of the clients in /usr/local/munki/preflight.d/cache/disk.plist is:

```xml
<?xml version="1.0" encoding="UTF-8"?>

<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">

<plist version="1.0">

<array>

	<dict>

		<key>BusProtocol</key>

		<string>SATA</string>

		<key>Content</key>

		<string>Apple_HFS</string>

		<key>Ejectable</key>

		<false/>

		<key>FreeSpace</key>

		<integer>138600611840</integer>

		<key>Internal</key>

		<true/>

		<key>MountPoint</key>

		<string>/</string>

		<key>RAIDMaster</key>

		<false/>

		<key>SMARTStatus</key>

		<string>Verified</string>

		<key>SolidState</key>

		<true/>

		<key>TotalSize</key>

		<integer>511250432000</integer>

		<key>VolumeName</key>

		<string>Macintosh HD</string>

	</dict>

	<dict>

		<key>BusProtocol</key>

		<string>USB</string>

		<key>Content</key>

		<string>Apple_HFS</string>

		<key>Ejectable</key>

		<true/>

		<key>FreeSpace</key>

		<integer>565001850880</integer>

		<key>Internal</key>

		<false/>

		<key>MountPoint</key>

		<string>/Volumes/WorkDump</string>

		<key>RAIDMaster</key>

		<false/>

		<key>SMARTStatus</key>

		<string>Not Supported</string>

		<key>TotalSize</key>

		<integer>999860912128</integer>

		<key>VolumeName</key>

		<string>WorkDump</string>

	</dict>

</array>

</plist>
```

**Comment by bochoven** [comment]
Should be fixed with f1aa8b34e53328da1757bd1db7afc8222d392c7a

**Comment by poundbangbash** [comment]
Confirmed that this corrects the disk size data when viewing the "Disk" listing.
